frontend/src/app/home/Connected.tsx

```diff
@@ -41,6 +42,7 @@ export function Connected() {
 
   return (
     <div className="flex flex-col gap-3 p-3">
+      {network?.chainId !== "2" && <Modal />}
```

Comment:
nit: can we add a `const TEST_NET = "2"` and check `network?.chainId !== TEST_NET`

Reply:
done